uaclient/tests/test_status.py

```diff
@@ -140,3 +140,28 @@ def test_no_leading_newline(
             status_dict = status_dict_unattached
 
         assert not format_tabular(status_dict).startswith("\n")
+
+    @pytest.mark.parametrize(
+        "user_facing_details, uf_status, uf_descr",
+        (
+            ("", "n/a", "Common Criteria EAL2 default descr"),
+            ("Custom descr", "n/a", "Custom descr"),
+            ("Custom call to action", "enabled", "Custom call to action"),
+        ),
+    )
+    def test_custom_descr(
+        self, user_facing_details, uf_status, uf_descr, status_dict_attached
+    ):
+        """Services can provide a custom call to action if present."""
+        default_descr = "Common Criteria EAL2 default descr"
+        status_dict_attached["services"] = [
+            {
+                "name": "cc-eal",
+                "description": default_descr,
+                "available": "no",
+                "status": uf_status,
+                "entitled": True,
+                "userFacingDetails": user_facing_details,
```

Comment:
Seeing this dict written out like this makes me realised that `userFacingDetails` is inconsistently named compared to the other keys. (For starters, practically by definition, anything in this dict is user-facing. :grin:)

Could we find a different key name?

Reply:
good point. how about customDescription.

Reply:
I'd stick with underscored key names for a couple of reasons: firstly, I find them more aesthetically pleasing but secondly, and this might actually be a good reason, it'll give us a heuristic for knowing if we're dealing with JSON we generated vs JSON returned from the contract server.

`custom_description` sounds good to me, though I wonder if we can be more specific than "custom". Perhaps `description_override` is more descriptive of how we use it? Naming things is hard, I'll leave it to your discretion.